src/components/DigitalCredentials/CredentialsStepper.vue

```diff
@@ -0,0 +1,237 @@
+                Your Business Card is now ready to use
+              </p>
+            </v-card-text>
+          </v-card>
+        </v-col>
+      </v-row>
+    </v-container>
+    <CredentialsWebSocket
+      @onConnection="handleConnection"
+      @onIssuedCredential="handleIssuedCredential"
+    />
+  </section>
+</template>
+
+<script lang="ts">
+import { DigitalCredentialTypes, Routes } from '@/enums'
+import { LegalServices } from '@/services'
+import { useBusinessStore } from '@/stores'
+import { Getter } from 'pinia-class'
+import { Component, Vue } from 'vue-property-decorator'
+import QrcodeVue from 'qrcode.vue'
+import { DigitalCredentialIF, WalletConnectionIF } from '@/interfaces'
+import CredentialsWebSocket from '@/components/DigitalCredentials/CredentialsWebSocket.vue'
+
+Component.registerHooks(['beforeRouteEnter'])
+
+// Create a component that extends the Vue class called CredentialsStepper
+@Component({ components: { QrcodeVue, CredentialsWebSocket } })
+export default class CredentialsStepper extends Vue {
+  @Getter(useBusinessStore) getIdentifier!: string;
+
+  credentialTypes = DigitalCredentialTypes;
+  credentialConnection: WalletConnectionIF = null;
+  issuedCredential: DigitalCredentialIF = null;
+
+  async beforeRouteEnter (to, from, next): Promise<void> {
+    next(async (_this) => {
+      const { data } = await LegalServices.fetchCredentials(_this.getIdentifier)
+      const issuedCredentials = data?.issuedCredentials
+
+      if (issuedCredentials?.length) {
+        next({ name: Routes.DIGITAL_CREDENTIALS })
+      } else {
+        next()
+      }
+    })
+  }
+
+  async mounted (): Promise<void> {
+    await this.getCredentialsConnection()
+    if (!this.credentialConnection) {
+      await this.addCredentialInvitation()
+    }
+  }
+
+  async addCredentialInvitation (): Promise<void> {
+    const { data: connection } = await LegalServices.createCredentialInvitation(this.getIdentifier)
+    this.credentialConnection = connection
+  }
+
+  async getCredentialsConnection (): Promise<void> {
+    const { data } = await LegalServices.fetchCredentialConnections(this.getIdentifier)
+    this.credentialConnection = data?.connections?.[0]
+  }
+
+  async issueCredential (credentialType: DigitalCredentialTypes): Promise<void> {
+    await LegalServices.sendCredentialOffer(this.getIdentifier, credentialType)
+  }
+
+  async handleConnection (connection: WalletConnectionIF) {
+    this.credentialConnection = connection
+    await this.issueCredential(this.credentialTypes.BUSINESS)
+  }
+
+  async handleIssuedCredential (issuedCredential: DigitalCredentialIF) {
+    this.issuedCredential = issuedCredential
+  }
+}
+</script>
+
+<style lang="scss" scoped>
+.credentials-stepper-number {
+  color: rgb(255, 255, 255);
+  background-color: rgb(22, 105, 187);
+}
+</style>
+
+<style scoped lang="scss">
+.word-wrap {
+  word-break: normal;
+}
+</style>
```

Comment:
Can you combine these style blocks?

For this particular class,  you may want to add "word-break-normal" in base.scss instead.

Reply:
Looks good.